backend/db/sql/schema.sql

```diff
@@ -7,13 +7,14 @@ USE literacy_link_db;
 
 CREATE TABLE IF NOT EXISTS Users (
      user_id BINARY(16) DEFAULT (UUID_TO_BIN(UUID(), 1)) PRIMARY KEY,
-     username VARCHAR(255) NOT NULL,
+     username VARCHAR(255) NOT NULL UNIQUE,
      password_hash VARCHAR(255) NOT NULL,
-     email VARCHAR(255) NOT NULL,
-     phone_number VARCHAR(20),
+     email VARCHAR(255) NOT NULL UNIQUE,
+     phone_number VARCHAR(20) NOT NULL UNIQUE,
      first_name VARCHAR(50) NOT NULL,
      last_name VARCHAR(50) NOT NULL,
      mailing_address VARCHAR(255),
+     pref_method_comm VARCHAR(10), -- Has to be "C" for call, "T" for text, or "E" for email.
```

Comment:
think there might be a way to add a constraint to the db to enforce this but we can add that later.

Reply:
Looks like we can use check:
https://www.w3schools.com/mysql/mysql_check.asp